# VIN information as reported by vehicle 

std_msgs/Header header

char vis_10   # Vehicle Identifier Section 10         
char vis_11   # Vehicle Identifier Section 11               
char vis_12   # Vehicle Identifier Section 12         
char vis_13   # Vehicle Identifier Section 13         
char vis_14   # Vehicle Identifier Section 14         
char vis_15   # Vehicle Identifier Section 15         
char vis_16   # Vehicle Identifier Section 16         
char vis_17   # Vehicle Identifier Section 17         
